### PR TITLE
Documentation and tests for ViewModelBase

### PR DESCRIPTION
**Pull request by SavikPavel**
### Description

Documentation and tests for ViewModelBase added

### API Changes
 
 None

### Platforms Affected

- Core (all platforms)

### Behavioral/Visual Changes

None

### Before/After Screenshots

Not applicable

### PR Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/Softeq/XToolkit.WhiteLabel/blob/master/.github/CONTRIBUTING.md) document
- [x] My code follows the [code styles](https://github.com/Softeq/dotnet-guidelines)
- [x] Targets the correct branch
- [x] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: Softeq.XToolkit.WhiteLabel/Mvvm/ViewModelBase.cs [hunk 1]
```diff
@@ -5,39 +5,49 @@
...
 
+        /// <summary>
+        ///     Gets a value indicating whether this ViewModel is initialized.
+        /// </summary>
         public bool IsInitialized { get; private set; }
 
+        /// <summary>
+        ///     Called when the page or UI component corresponding to this ViewModel
+        ///     is initializing.
+        /// </summary>
```
Comment (wcoder) [inline Softeq.XToolkit.WhiteLabel/Mvvm/ViewModelBase.cs:33]: Could you please move docs to the interface and use `<inheritdoc>` here?

File: Softeq.XToolkit.WhiteLabel.Tests/Mvvm/ViewModelBaseTests/ViewModelBaseTests.cs [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        [InlineData(true, false)]
+        [InlineData(false, true)]
+        public void ViewModelBase_NotifiesWhenIsBusyChanges(bool initialIsBusy, bool changedIsBusy)
+        {
+            var vm = new ViewModelStub()
+            {
+                IsBusy = initialIsBusy
+            };
+
+            var raisedEvent = Assert.Raises<PropertyChangedEventArgs>(
```
Comment (wcoder) [inline Softeq.XToolkit.WhiteLabel.Tests/Mvvm/ViewModelBaseTests/ViewModelBaseTests.cs:42]: Can be used `Assert.PropertyChanged()`

File: Softeq.XToolkit.WhiteLabel/Mvvm/ViewModelBase.cs (hunk 1 above)
Comment (SavikPavel) [reply Softeq.XToolkit.WhiteLabel/Mvvm/ViewModelBase.cs]: Sure thing, done!

File: Softeq.XToolkit.WhiteLabel.Tests/Mvvm/ViewModelBaseTests/ViewModelBaseTests.cs (hunk 2 above)
Comment (SavikPavel) [reply Softeq.XToolkit.WhiteLabel.Tests/Mvvm/ViewModelBaseTests/ViewModelBaseTests.cs]: Fixed

**Review by wcoder — APPROVED** [review]

**Pull request merged**